### PR TITLE
fix msisdn in s11 interface

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/s11/handlers/create_session_handler.c [hunk 1]
```diff
@@ -156,7 +156,8 @@ create_session_processing()
 	ies[ieCount].type = IE_MSISDN;
 	ies[ieCount].length = htons(5);
 	ies[ieCount].instance = INSTANCE_ZERO;
-	memcpy(ies[ieCount].value, g_csReqInfo->MSISDN, 5);
+	//memcpy(ies[ieCount].value, g_csReqInfo->MSISDN, 5);
```
Comment (krsna1729) [inline src/s11/handlers/create_session_handler.c]: Why not delete this if its wrong?

**Review by krsna1729 — COMMENTED** [review]
Need commit message

**Comment by ghost** [comment]
please retest

**Comment by micpez** [comment]
retest this please

File: src/s11/handlers/create_session_handler.c (hunk 1 above)
Comment (ghost) [reply src/s11/handlers/create_session_handler.c]: solved in second patch

**Comment by badhrinathpa** [comment]
+1

**Comment by krsna1729** [comment]
squash the commits

**Review by vbarate — APPROVED** [review]

**Pull request merged**